static/sass/patterns/_forms.scss

```diff
@@ -25,25 +25,3 @@
 .form-actions {
   text-align: right;
 }
-
-.btn {
-  &,
-  #{$link} {
-    @include contrasted('action');
-    background-image: none; // remove link underline when styling as button...
-    border-color: transparent;
-    border-style: solid;
-    border-width: 1px;
-    display: inline-block;
-    padding: 0.5rem 1rem;
-    text-decoration: none;
-    text-shadow: none;
-    transition: color 150ms, background-color 150ms;
```

Comment:
I don't follow, sorry.

There were a number of additions to this block in the PR we merged this morning (see link above). They're removed again in this branch, and it happened in a merge resolution. Wanted to flag that, because I'm guessing we want to re-add those to the new `_buttons.scss` file.